analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CommandPath.cs

```diff
@@ -1,8 +1,130 @@
-using System;
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security;
```

Comment:
could you also create your own "ProcessStartInfo" class in a different namespace and use that?